sway/desktop/render.c

```diff
@@ -420,29 +421,40 @@ static void render_titlebar(struct sway_output *output,
 		render_rect(output->wlr_output, output_damage, &box, color);
 	}
 
-	size_t inner_width = width - titlebar_h_padding * 2;
+	int inner_x = x - output_x + titlebar_h_padding;
 	int bg_y = y + titlebar_border_thickness;
+	size_t inner_width = width - titlebar_h_padding * 2;
+
+	int ob_inner_x = round(inner_x * output_scale);
```

Comment:
Can we add `// output-buffer-local` comments for those two?

Reply:
Done